### PR TITLE
Naveedn bugfix broken urls

### PR DESCRIPTION
**Pull request by naveedn**
/documentation

#### Which issue(s) this PR fixes:

Fixes the three cases identified in this comment: https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/issues/374#issuecomment-1452287703. 

Should address issues:
-  #374
-  #369  
- #386

### CODE REVIEW

**Comment by naveedn** [comment]
I wish I had known about https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/issues/349, I think that enhancement makes sense -- this ticket would have more straightforward with some knowledge a priori.

File: utils/generate-docs/Schema/SchemaReader.ts [hunk 1]
```diff
@@ -23,9 +23,9 @@ export default class SchemaReader {
   protected loadJsonFromEntryInfo = (
     entryInfo: EntryInfo
   ): Promise<SchemaNodeJson> =>
-    import(pathToFileURL(entryInfo.fullPath).href).then(
-      (json: { default: SchemaNodeJson }) => json["default"]
-    );
+    import(pathToFileURL(entryInfo.fullPath).href, {
```
Comment (JSv4) [inline utils/generate-docs/Schema/SchemaReader.ts:26]: This change is throwing a rather cryptic syntax error for me (and is breaking the test too):

```console
jman@jman-Blade-15-Advanced-Model-Mid-2021-RZ09-0409:~/source/Open-Cap-Format-OCF$ npm run generate-docs

> generate-docs
> node --loader ts-node/esm --no-warnings --experimental-json-modules ./utils/generate-docs/index.ts

SyntaxError: Unexpected token ','
    at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:139:18)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:236:14)
```

Based on what you added, my guess is one of our jsons has a trailing comma or something that one validator likes and another doesn't or something like that? Your change to `PathTools.ts` works. I'm inclined to drop the assertions and move resolving whatever is causing this error to a separate, low priority task.

**Review by JSv4 — CHANGES_REQUESTED** [review]
Thanks for doing this @naveedn. I know what a pain working out the paths for mkdocs can be. Your fix makes sense and should work. I'm seeing an issue with the tests that seems to have been introduced by your addition of `assert: {type: "json"}` in `SchemaReader.ts` and `ExampleReader.ts`. I suspect that, for some reason, a file of ours is failing the TypeScript / node validation for type json but is passing other validators. Unless you have a quick fix for this (there's not much to go off of based on the error I got), I'd propose removing those changes, merging your changes to `PathTools.ts`, and opening a new issue to figure out why your assertions fail.

File: utils/generate-docs/Schema/SchemaReader.ts (hunk 1 above)
Comment (JSv4) [reply utils/generate-docs/Schema/SchemaReader.ts:26]: Just saw #386 . Fix here is upgrading to node v16.16.0

Comment (JSv4) [reply utils/generate-docs/Schema/SchemaReader.ts:26]: Node 16 is almost end of life. Security support ends September of this year. We should upgrade to 18 LTS.

**Comment by JSv4** [comment]
@pjohnmeyer or @jacobyavis, can one of you approve this? I upgraded the node version and now can't approve the PR as I am an author now too. D'oh.

**Review by JSv4 — APPROVED** [review]

**Comment by JSv4** [comment]
Never mind... just needed to refresh the page, I guess.

**Pull request merged**